src/System.Dynamic.Runtime/ref/System.Dynamic.Runtime.Manual.cs

```diff
@@ -6,12 +6,3 @@
 // ------------------------------------------------------------------------------
 
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.ConditionalWeakTable<,>))]
```

Comment:
You can go ahead and remove this manual file and move the one type-forward to the other file.